test/test_examples_2d.jl

```diff
@@ -97,104 +103,134 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
+      linf = [0.11749257043751615, 0.7223475657303381, 0.7223475657303381, 5.425015419074852],
+      maxiters=1, initial_condition=initial_condition_medium_sedov_blast_wave)
   end
 
+  @testset "taal-check-me elixir_euler_blob_shockcapturing_amr.jl" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_blob_shockcapturing_amr.jl"),
+      l2   = [0.2012143467980036, 1.1813241716700988, 0.10144725208346557, 5.230607564921326],
+      linf = [14.111578610092542, 71.21944410118338, 7.304666476530256, 291.9385076318331],
+      tspan = (0.0, 0.12))
+  end
+
+  @testset "taal-check-me elixir_euler_khi_shockcapturing.jl" begin
+    if Threads.nthreads() == 1
+      # This example uses random numbers to generate the initial condition.
+      # Hence, we can only check "errors" if everything is made reproducible.
+      # However, that's not enough to ensure reproducibility since the stream
+      # of random numbers is not guaranteed to be the same across different
+      # minor versions of Julia.
+      # See https://github.com/trixi-framework/Trixi.jl/issues/232#issuecomment-709738400
+      test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_khi_shockcapturing.jl"),
+        l2   = [0.0020460050625351277, 0.0028624298590723372, 0.001971035381754319, 0.004814883331768111],
+        linf = [0.02437585564403255, 0.018033033465721604, 0.00993916546672498, 0.02097263472404709],
+        tspan = (0.0, 0.2))
+    else
+      test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_khi_shockcapturing.jl"),
+        tspan = (0.0, 0.2))
+    end
+  end
+
+  @testset "taal-check-me elixir_euler_khi_shockcapturing_amr.jl" begin
+    if Threads.nthreads() == 1
+      # This example uses random numbers to generate the initial condition.
+      # Hence, we can only check "errors" if everything is made reproducible.
+      # However, that's not enough to ensure reproducibility since the stream
+      # of random numbers is not guaranteed to be the same across different
+      # minor versions of Julia.
+      # See https://github.com/trixi-framework/Trixi.jl/issues/232#issuecomment-709738400
+      test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_khi_shockcapturing_amr.jl"),
+        l2   = [0.001617236176233394, 0.0023394729603446697, 0.001296199247911843, 0.0033150160736185323],
+        linf = [0.019002843896656074, 0.017242107049387223, 0.008179888370650977, 0.016885672229959958],
+        tspan = (0.0, 0.2))
+    else
+      test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_khi_shockcapturing_amr.jl"),
+        tspan = (0.0, 0.2))
+    end
+  end
+  
   @testset "taal-check-me elixir_euler_vortex.jl" begin
   test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_vortex.jl"),
     l2   = [3.6342636871275523e-6, 0.0032111366825032443, 0.0032111479254594345, 0.004545714785045611],
     linf = [7.903587114788113e-5, 0.030561314311228993, 0.030502600162385596, 0.042876297246817074])
   end
-
+  
```

Comment:
```suggestion

```
Could you please try not to commit lines containing only whitespace? Shall we have some git hook to prevent this automatically?

Reply:
Sorry, my bad. I don't think there's a hook we can install on GitHub, can we? But maybe it would be a good idea to provide a set of useful hooks we can ask Trixi devs to install locally (eg, also preventing adding large files etc)